src/stories/Library/slider/init-slider.js

```diff
@@ -43,10 +45,9 @@ function swiperWrapperEventInit(swiperWrapper) {
     // Calculate a new translate value, for pulling the slider.
     // We add half of the latest known sidemargin, so the active slide doesn't
     // go flush against the screen.
-    // eslint-disable-next-line no-param-reassign
-    swiperWrapper.style.transform = `translate3d(${Math.floor(
-      translateWidth + sideMargins / 2
-    )}px, 0, 0)`;
+    translateWidth = Math.floor(translateWidth + sideMargins / 2);
+
+    swiper.setTranslate(translateWidth);
```

Comment:
:+1: Nice to use Swiper API for this.

Reply:
yeah, i dont know why it didnt work before - i tried it before, when you suggested it.